components/arm/xarm/xarm.go

```diff
@@ -251,9 +251,33 @@ func (x *xArm) DoCommand(ctx context.Context, cmd map[string]interface{}) (map[s
 		if err != nil {
 			return nil, err
 		}
+		return map[string]interface{}{}, nil
```

Comment:
I've added this since we were returning an error on `move_gripper` when there was none in actuality.

Reply:
This should be solved differently. Instead, a flag should be set if at least one command is run. We do not want to return per-command, or else you cannot set speed and acceleration simultaneously.

Reply:
lmk what you think of the current solution.